internal/querynodev2/segments/retrieve.go

```diff
@@ -65,10 +66,19 @@ func retrieveOnSegments(ctx context.Context, mgr *Manager, segments []Segment, s
 		wg.Add(1)
 		go func(seg Segment, i int) {
 			defer wg.Done()
-
+			// record search time and cache miss
```

Comment:
Maybe it would be better to stats cache access in Do method of DiskCache?

Reply:
SegmentAccess related metric is not bound with disk cache.
Segment without disk cache still have segment access metric.